### PR TITLE
8.3.0a

### PR DESCRIPTION
**Pull request by drewbratcher**
Updates our fork to the latest version of Dropbox. 
Last update was 7.0.0
See https://github.com/Ginger-Labs/Notability/issues/7259 for Notability issue
See https://github.com/dropbox/SwiftyDropbox/releases for release notes

A separate PR will be required to use this updated version in the Notability repo

### CODE REVIEW

**Comment by drewbratcher** [comment]
I attempted to get their unit tests to run, but I found the instructions pretty time consuming sounding:
```
        // You need an API app with the "Full Dropbox" permission type and at least the scopes in DropboxTester.scopes
        // and no team scopes.
        // You can create one for testing here: https://www.dropbox.com/developers/apps/create
        // The 'App key' will be on the app's info page.
        // Then follow https://dropbox.tech/developers/pkce--what-and-why- to get a refresh token using the PKCE flow
```

let me know if you think it's worth pursuing though

File: Source/SwiftyDropbox/Shared/Generated/Contacts.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ open class Contacts {
         /// List of manually added contacts to be deleted.
         public let emailAddresses: Array<String>
         public init(emailAddresses: Array<String>) {
-            arrayValidator(itemValidator: stringValidator(maxLength: 255, pattern: "^['&A-Za-z0-9._%+-]+@[A-Za-z0-9-][A-Za-z0-9.-]*\\.[A-Za-z]{2,15}$"))(emailAddresses)
+            arrayValidator(itemValidator: stringValidator(maxLength: 255, pattern: "^['#&A-Za-z0-9._%+-]+@[A-Za-z0-9-][A-Za-z0-9.-]*\\.[A-Za-z]{2,15}$"))(emailAddresses)
```
Comment (kydonnelly) [inline Source/SwiftyDropbox/Shared/Generated/Contacts.swift:16]: good ol' email regex

**Review by kydonnelly — APPROVED** [review]
Looks like the main thing to look out for is replacing deprecated calls V2, callback queues, and maybe something about team status

**Comment by kydonnelly** [comment]
Are there any open issues for this version?

**Comment by drewbratcher** [comment]
They have about [40 open issues,](https://github.com/dropbox/SwiftyDropbox/issues) but all stem from prior releases. I browsed through and didn't see anything particularly worrying.

**Comment by kydonnelly** [comment]
> They have about [40 open issues,](https://github.com/dropbox/SwiftyDropbox/issues) but all stem from prior releases. I browsed through and didn't see anything particularly worrying.

Thanks for taking a look! "Alamofire: But Why?" is my favorite

**Comment by drewbratcher** [comment]
lol me too, especially since the Alamofire upgrade is being such a pain for me right now

**Pull request closed without merging**